### PR TITLE
feat(JS rules): add expressjs rule for missing HTTPS protocol

### PR DESCRIPTION
**Pull request by elsapet**
## Description
Add rule to trigger in case an app is using `http.createServer()` and not `https.createServer()`

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [ ] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Review by cfabianski — APPROVED** [review]

File: pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - javascript
+trigger: absence
+severity:
+  default: "medium"
+metadata:
+  description: "Security misconfiguration detected."
+  remediation_message: |
+    ## Description
+
+    Helmet can help protect your app from some well-known web vulnerabilities by setting HTTP headers appropriately.
```
Comment (gotbadger) [inline pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml]: Description seems wrong here?

Comment (elsapet) [reply pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml]: 🙈

Comment (elsapet) [reply pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml]: Thanks for the catch @gotbadger - updated 👍

Comment (cfabianski) [reply pkg/commands/process/settings/rules/javascript/express/https_protocol_missing.yml]: Nice catch @gotbadger

**Pull request merged**